cloud/logger/src/index.ts

```diff
@@ -0,0 +1,77 @@
+import Logger from './Logger';
+import {LOG_LEVEL} from './Logger';
+
+/**
+ * Raincatcher logger implementation, can take a different logger at construction time.
+ */
+export default class LoggerImpl implements Logger {
+  private logger: any;
```

Comment:
-1 from me, I'd rather have our logger module implement the interface on top of a specific lib like bunyan/debug and leave others for upstream contribution through something like `@raincatcher/logger-debug`.

The semantics for most of the libs around these parameters are very different, so just forwarding the arguments likely won't work.

i.e. for bunyan we have to add the extra data to the object we pass to it, and for debug the tags would be prefixes and so on.